configs/config.d/digauth.cf

```diff
@@ -0,0 +1,15 @@
+#
+# Configure remote debugging for the cache server for the OSG Data Federation.
+# To enable - uncomment/adjust last line(s) when there is a problem with the server.
+#
+# ************************************************************
+# * NOTE: Changes to this file will be PRESERVED on upgrades *
+# ************************************************************
+#
+# This file is part of the StashCache Daemon
+# https://opensciencegrid.github.io/StashCache-Daemon
+#
+#all allow host h=login01.osgconnect.mwt2.org
+#all allow host h=*.unl.edu
+#all allow gsi g=cms n=theuser 
```

Comment:
For `g=cms`: I believe `libXrdLcmaps.so` maps VOMS group names directly to Xrootd group names.  So, the example is probably `/cms`.

For `n=theuser` -> how about give a Unix username that is in the default LCMAPS mappings?

Reply:
1. Can these be ANDed together, e.g. only allow `/cms` people from UNL?
2. Could you add comments explaining what these settings mean? Especially `n=theuser`, which the manual says is "authentication-specific"
3. Maybe a line near the top explaining what information this provides -- something like
```
# This allows remote systems to view the XRootD config, core files,
# logs, and process information, for debugging purposes.
# Since this information is sensitive, limit access to only those that
# need it, and turn it on only when remote debugging is required.
```

Reply:
OK, I'll add to it more ... We also don't want to rewrite xrootd doc which is already telling how to use this, that's why I thought linking original documentation in the config might give a clue. Also, I haven't tried other examples than `host`-based auth so I wouldn't be confident provide other examples which may not work (we can still test it though, but that's not a priority atm). Overall, we wanted just give a hint this exist and ship simple config file according to [SOFTWARE-3508](https://opensciencegrid.atlassian.net/browse/SOFTWARE-3508).

Reply:
Yeah it should be more of a 'reminder' than a rewrite.

Reply:
I've added what I knew applies and should work. Not sure if that's how you like it, though. If unhappy I'd need more help on expanding the knowledge on digauth from developers.

Reply:
What does the `n=theuser` do?

Reply:
added explanation:
```
# g=/cms    : VOMS group name mapped to XRootD group name
# n=cmsprod : unix username that is in the default LCMAPS mappings
```
sounds good?

Reply:
Sure.